src/modules/fundingManager/bondingCurve/abstracts/BondingCurveBase_v1.sol

```diff
@@ -322,14 +322,17 @@ abstract contract BondingCurveBase_v1 is IBondingCurveBase_v1, Module_v1 {
             _getFeeManagerIssuanceFeeData(_selector);
     }
 
-    /// @dev Calculates the propotion of the fees for the given amount and returns them plus the amount minus the fees
+    /// @dev    Calculates the propotion of the fees for the given amount and returns them plus the amount minus the fees
```

Comment:
`propotion` -> `proportion`
(i know you didnt do this, but lets still fix that I think